client/components/loading-spinner/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
+
+const LoadingSpinner = ( { inline } ) => {
```

Comment:
Nope. We are using spinners everywhere, not only in buttons. Eventually we'll need to switch to using the cool "pulsating placeholder" Calypso pattern, but that's out of the scope of this PR.